docs/navigation-configuration.md

```diff
@@ -34,6 +34,11 @@ A sample navigation structure looks as follows:
         label: 'Home',
         viewUrl: 'https://my.microfrontend.com/',
         children: [
+          {
+            absolutePath: '/home',
+            label: 'Go back home',
+            viewUrl: 'https://my.microfrontend.com/'
```

Comment:
As I wrote in the configuration file, the `viewUrl` is not needed.

Reply:
Removed